src/components/Sidebar/index.tsx

```diff
@@ -6,64 +6,62 @@ import { ReactComponent as CloseSvg } from 'assets/icons/close.svg';
 import { ReactComponent as SelectorSvg } from 'assets/icons/selector.svg';
 
 // Components
-import {
-    StyledCloseContainer,
-    StyledLink,
-    StyledNav,
-    StyledSidebarBackground,
-} from './components';
+import { StyledCloseContainer, StyledLink, StyledNav, StyledSidebarBackground } from './components';
 
 interface ISidebar {
     setSidebar: React.Dispatch<React.SetStateAction<boolean>>;
     sidebar: boolean;
 }
-const Sidebar: React.FC<ISidebar> = ({
-    setSidebar,
-    sidebar,
-}) => {
+const Sidebar: React.FC<ISidebar> = ({ setSidebar, sidebar }) => {
     return (
         <StyledSidebarBackground active={sidebar}>
             <StyledCloseContainer onClick={() => setSidebar(false)}>
                 <CloseSvg />
             </StyledCloseContainer>
             <StyledNav>
                 <ul>
-                <li>
-                    <div style={{ height: '22px', width: '22px' }}>
-                        <SelectorSvg fill={window.location.pathname === '/propose' ? colors.yellow : 'transparent'} />
-                    </div>
-                    <StyledLink onClick={() => setSidebar(false)} exact activeClassName='active' to='/propose'>
-                        Propose App
-                    </StyledLink>
-                </li>
-                <li>
-                    <div style={{ height: '22px', width: '22px' }}>
-                        <SelectorSvg fill={window.location.pathname === '/new-farm' ? colors.yellow : 'transparent'} />
-                    </div>
-                    <StyledLink onClick={() => setSidebar(false)} exact activeClassName='active' to='/new-farm'>
-                        New Farm
-                    </StyledLink>
-                </li>
-                <li>
-                    <div style={{ height: '22px', width: '22px' }}>
-                        <SelectorSvg fill={window.location.pathname === '/stats' ? colors.yellow : 'transparent'} />
-                    </div>
-                    <StyledLink onClick={() => setSidebar(false)} exact activeClassName='active' to='/stats'>
-                        Stats
-                    </StyledLink>
-                </li>
-                <li>
-                    <div style={{ height: '22px', width: '22px' }}>
-                        <SelectorSvg fill={window.location.pathname === '/my-farm' ? colors.yellow : 'transparent'} />
-                    </div>
-                    <StyledLink onClick={() => setSidebar(false)} exact activeClassName='active' to='/my-farm'>
-                        My Farm
-                    </StyledLink>
-                </li>
+                    <li>
+                        <div style={{ height: '22px', width: '22px' }}>
```

Comment:
ditto with the css prop and using `px`s

Reply:
Fixed in 71b12631f095a0169dd86908bf04935aeec207ef